fastapi_websocket_pubsub/event_broadcaster.py

```diff
@@ -78,46 +149,39 @@ async def __broadcast_notifications__(self, subscription: Subscription, data):
             data: the event data
         """
         logger.info("Broadcasting incoming event",
-                    {'topic':subscription.topic, 'notifier_id':self._id})
+                    {'topic': subscription.topic, 'notifier_id': self._id})
         note = BroadcastNotification(notifier_id=self._id, topics=[
                                      subscription.topic], data=data)
         # Publish event to broadcast
         async with self._publish_lock:
             async with self._broadcast:
                 await self._broadcast.publish(self._channel, note.json())
 
+    def _acquire_publishing_broadcast(self):
+        self._broadcast = self._broadcast_type(self._broadcast_url)
+
+    async def _subscribe_to_all_topics(self):
+        return await self._notifier.subscribe(self._id,
+                                              ALL_TOPICS,
+                                              self.__broadcast_notifications__)
+
+    async def _unsubscribe_from_topics(self):
```

Comment:
_unsubscribe_from_all_topics (to be more symmetric).

even better:
_subscribe_to_all_topics -> _start_broadcasting_incoming_messages_from_all_topics
_unsubscribe_from_topics -> _stop_broadcasting_incoming_messages_from_all_topics

Reply:
The function removes all subscriptions, rather than the subscription from ALL_TOPICS  

"start_broadcasting_incoming_messages_from_all_topics" LOL is that a name for a function, or the first chapter of a book ?

Reply:
i think it's more descriptive, but only a suggestion.

at any rate, make the functions same convention (i.e: if you stick with _subscribe_to_all_topics, make the other one _unsubscribe_from_all_topics)